stage-process.md

```diff
@@ -0,0 +1,115 @@
+
+  * What's needed from individuals?
+    * [ ] Typically, a detailed (but IP-safe, high-level) issue describing a user story or problem
+        * Optionally, a personal repo, a blog post with examples, a mock-up, etc. 
+  * Who is involved?
+    * CG meeting time should be minimally requested except in the form of "announcements"
+    * CG Mailing List is an appropriate place for high-level or user-story discussion.
+    * It is also recommended that community venues like [SocialHub](https://socialhub.activitypub.rocks/), [Indieweb.org Brainstorming](https://indieweb.org/wiki#Brainstorming), [Fediverse Ideas](https://codeberg.org/fediverse/fediverse-ideas/), or the social web itself be used to gather more input and to poll interest.
+    * Descriptive specifications of a prototype (whether as a drafting exercise or as an invitation to prototype interoperability) can be hosted on git forges, including community venues like [the Fediverse Enhancement Proposals](https://codeberg.org/fediverse/fep/)
+  * What's the optimal outcome?
+    * Objective, open description of a problem or user story [set] that is within scope of the CG
+    * A champion to gather and synthesis feedback, from outside the CG and to coordinate with the CG for review and future stages of work
+
+## Stage 1: Incubation (Community and/or Community Group)
+
+The purpose of Stage 1 proposals is to explore the Solution Space and evaluate multiple candidate solutions fairly.
+This is where most of the work gets done because it involves exploring alternatives, understanding tradeoffs, gathering implementation experience, incubating alternatives, gathering evidence of demand and fitness for purpose, and finally, identifying the best out of the many alternatives.
+
+**Great care** should be taken to avoid substantial technical/design discussions happening outside the "IPR" boundary of the Community Group members if normative status beyond stage 1 is desired;
+further standardization MUST be rejected if "clean IPR" cannot be achieved (e.g. by post-facto IP grants from all substantial contributors).
+
+  * What's needed from champions?
+    * [ ] Public identification of one or more [champions](https://github.com/tc39/how-we-work/blob/main/champion.md)
+    * [ ] An understanding of the problem space (need not be *deeply* technical)
+    * [ ] Familiarity with prior art and related parallel work
+  * What's asked of the **Community Group**?
+    * [ ] The Community Group consensus that the problem is worth spending the Community Group’s time working on
+    * [ ] A home for incubating the proposal. Small decisions will incubate in tracking issues. If and when the champions are ready to more thoroughly document their proposal, the WG chairs will create a repo for the champions to develop a cohesive proposal (e.g., [example](https://github.com/swicg/activitypub-http-signature)). 
```

Comment:
```suggestion
    * [ ] A home for incubating the proposal. Small features will incubate in issues. If and when the champions are ready to more thoroughly document their proposal, the WG chairs will create a repo for the champions to develop the feature or proposal (e.g., [example](https://github.com/swicg/activitypub-http-signature)). 
```

Revert from "small decisions" back to "small features" for reasons provided in today's Social CG call, and also revert "cohesive proposal" back to "feature", yet add "or proposal" since we do want to encourage incubation of both new features in existing specs and new proposal specs.

Reply:
👍 re reverting

Reply:
Accepting Tantek's change request (reverting 'decisions' back to original 'features' wording), per the discussion on the CG call today.